Formula/e/erlang.rb

```diff
@@ -73,6 +84,7 @@ def install
     system "./configure", *std_configure_args, *args
     system "make"
     system "make", "install"
+    system "./otp_build", "download_ex_doc"
```

Comment:
What is it downloading here? Can we avoid that with, e.g., `resource` blocks? We try not to have formulae download things during the build (and will block them from doing so at some point).

Reply:
This is what `download_ex_doc` does: https://github.com/erlang/otp/blob/57dbdfe82c1aa55ecf6e4a6f6f758d98e84abd24/otp_build#L1013.

If it's possible to do with a `resources` block I can do it, but otherwise this is Erlang/OTP specific, and may change with build scripts, so don't know how easy it'd be to maintain here.

Reply:
There's already a `resource` block in this formula. Below it, add,
```ruby
# https://github.com/erlang/otp/blob/#{version}/make/ex_doc_link
resource "ex_doc" do
  url "https://github.com/elixir-lang/ex_doc/releases/download/v0.34.1/ex_doc_otp_26"
  sha256 "d1e09ef6772132f36903fbb1c13d6972418b74ff2da71ab8e60fa3770fc56ec7"
end
```
and then in the `install` method, do
```
(buildpath/"bin").install resource("ex_doc") => "ex_doc"
```

Reply:
Thanks for this. I'll copy-and-paste it and adapt locally, if required.